src/NzbDrone.Core/Notifications/Discord/Discord.cs

```diff
@@ -715,15 +715,19 @@ private string GetTitle(Series series, List<Episode> episodes)
                 return null;
             }
 
+            if (episodes.Empty())
+            {
+                return series.Title.Replace("`", "\\`");
+            }
+
             if (series.SeriesType == SeriesTypes.Daily)
             {
                 var episode = episodes.First();
 
                 return $"{series.Title} - {episode.AirDate} - {episode.Title}".Replace("`", "\\`");
             }
 
-            var episodeNumbers = string.Concat(episodes.Select(e => e.EpisodeNumber)
-                                                       .Select(i => string.Format("x{0:00}", i)));
+            var episodeNumbers = string.Concat(episodes.Select(e => e.EpisodeNumber).Select(i => $"x{i:00}"));
```

Comment:
This can be done in one select instead of two:

```suggestion
            var episodeNumbers = string.Concat(episodes.Select(e => $"x{e.EpisodeNumber:00}"));
```